File: src/main/java/br/com/blu/bs2/sintetizador/contas/csv/service/validation/IValidadorService.java
package br.com.blu.bs2.sintetizador.contas.csv.service.validation;

import br.com.blu.bs2.sintetizador.contas.csv.utils.exceptions.ServiceException;

import java.io.File;
import java.util.List;

public interface IValidadorService {

    void validarPathEntradaArquivosCsv(String path) throws ServiceException;

    void validarContemArquivosObtidosFileDiretorioPadraoEntrada(List<File> fileInputDatList) throws ServiceException;
}

File: src/main/kotlin/br/com/blu/bs2/sintetizador/contas/csv/service/validation/IValidadorService.kt
package br.com.blu.bs2.sintetizador.contas.csv.service.validation

import br.com.blu.bs2.sintetizador.contas.csv.utils.exceptions.ServiceException
import java.io.File

interface IValidadorService {
    @Throws(ServiceException::class)
    fun validarPathEntradaArquivosCsv(path: String?)

    @Throws(ServiceException::class)
    fun validarContemArquivosObtidosFileDiretorioPadraoEntrada(fileInputDatList: List<File?>?)
}
